src/EventManagement.Domain/OrderLine.cs

```diff
@@ -39,7 +39,8 @@ public string ItemCode
 		{
 			get
 			{
-				return ProductVariantId.HasValue ? $"K{ProductId}-{ProductVariantId}" : $"K{ProductId}";
+                var prefix = IsRefund ? "R" : "K";
```

Comment:
Note that I need the product Id to be the same for refunded products and normal orderlines. the only difference would be the quantity